File: Algorithms/p171_Excel_Sheet_Column_Number/p171_Excel_Sheet_Column_Number.py
class Solution(object):
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        w = 1
        res = 0
        ord_A = ord('A')
        for i in range(len(s)-1,-1,-1):
            res += (ord(s[i])-ord_A+1)*w
            w *= 26
        return res

File: Algorithms/p171_Excel_Sheet_Column_Number/p171_Excel_Sheet_Column_Number.cpp
class Solution {
public:
    int titleToNumber(string s) {
        int res = 0;
        for(int i=s.size()-1,w=1;i>=0;i--){
            res += (s[i]-'A'+1)*w;
            w *= 26;
        }
        return res;
    }
};
